xml/System.Text/EncodingInfo.xml

```diff
@@ -331,7 +331,7 @@
   
  The <xref:System.Text.Encoding.GetEncodings%2A> method gets a complete list of supported encodings, uniquely distinguished by code page. If your application retrieves encodings using the <xref:System.Text.EncodingInfo.Name%2A> property, note that some duplicate encodings are retrieved. For more about handling these duplicates, see the description of <xref:System.Text.Encoding.GetEncodings%2A>.  
   
-   
+ For more information about the IANA, see <www.iana.org>.
```

Comment:
```suggestion
 For a list of supported names, see [Character Sets](https://www.iana.org/assignments/character-sets/character-sets.xhtml) on the [IANA website](https://www.iana.org).
```
Incidentally, the original `<www.iana.org>` doesn't resolve to a link, since it's missing `https://` or `http://`.